src/components/TimePicker/ampmSelect.js

```diff
@@ -71,6 +77,7 @@ export default class AmPmSelect extends PureComponent {
                         value="AM"
                         checked={this.isInputChecked('AM')}
                         onChange={this.handleOnChange}
+                        onClick={this.handleOnClick}
```

Comment:
I think here you can simply use the onClick prop direclty:
```onClick={onClick}```
then there is no need for:
```
this.handleOnClick = this.handleOnClick.bind(this);
...
handleOnClick() {
        const { onClick } = this.props;
        onClick();
    }
```

Reply:
Hi, @Leandro. I keep it that way because I was attempting to respect the project structure, but if you rather that way I could change it. Thanks for the comment :)

Reply:
Yes, we normally use handlers in this way but only when doing more things like for example:
```
handleOnClick() {
        const { onClick, disabled } = this.props;
        if (disabled) {
                     onClick();
        }
}
```
in a case like this we use to do like:
```onClick={onClick}```
since it is more simple and also avoids the binding which is costly

Reply:
Thanks for the clarification I'll change it 👍